### PR TITLE
Added some example playbooks

### PR DESCRIPTION
**Pull request by Tronde**
IMHO it's useful to have some example playbooks for common use cases that are easy to adapt by copy, paste, and modify.

I like it when roles have an example direktory with some playbooks in it, because it saves time when I don't have to search a long README.md for some examples.

With this PR I would like to contribute some example playbooks for the firewall role. I chose some use cases I encountered in different enviroments.

Signed-off-by: Joerg Kastning <jkastning@my-it-brain.de>

### CODE REVIEW

File: examples/disable-cockpit-and-dhcpv6-client.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+- name: Enable http and https service in default zone
```
Comment (richm) [inline examples/disable-cockpit-and-dhcpv6-client.yml:2]:
```suggestion
- name: Disable cockpit and dhcpv6-client service in default zone
```

**Comment by richm** [comment]
one minor nit, otherwise, lgtm

**Comment by Tronde** [comment]
Thanks for checking the PR. I fixed the name of the task in `example/disable-cockpit-and-dhcpv6-client.yml` as well as a `yamllint` error found by the tox check.

File: examples/enable-http-and-https.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+- name: Enable http and https service in default zone
+  hosts: "{{ targets }}"
+
+  vars:
+    firewall:
+      - service: [http,https]
```
Comment (richm) [inline examples/enable-http-and-https.yml:7]:
```suggestion
      - service: [http, https]
```

**Comment by Tronde** [comment]
Sorry, I missed that second issue with the missing space. I'll do better next time by trying to use the tools at hand in a proper way. As far as I could see I hope to have catched all yamllint issues and tox shoudn't find any others. Thanks for your patience.

**Comment by richm** [comment]
> Sorry, I missed that second issue with the missing space. I'll do better next time by trying to use the tools at hand in a proper way. As far as I could see I hope to have catched all yamllint issues and tox shoudn't find any others. Thanks for your patience.

Thank you for the PR!

**Review by richm — APPROVED** [review]

**Pull request merged**